wsd/LOOLWSD.cpp

```diff
@@ -1021,9 +1020,6 @@ void LOOLWSD::initialize(Application& self)
     // Allow UT to manipulate before using configuration values.
     UnitWSD::get().configure(config());
 
-    // Setup user interface mode
-    UserInterface = getConfigValue<std::string>(conf, "user_interface.mode", "classic");
```

Comment:
wait, this one goes from loolwsd.xml? as we discussed earlier also with @kendy: user_interface.mode should stay there. It setups the default mode if 'uidefaults' are not set for the request by integrator.

Reply:
In that place it is useless after we decided to activate notebookbar anyway. It was used only for activating notebookbar in core. The default mode is set here: https://github.com/CollaboraOnline/online/blob/private/gmasei/change_ui_mode/wsd/FileServer.cpp#L774 in case 'uiDefaults' are not set.

Reply:
ok then